EIPS/eip-7432.md

```diff
@@ -308,6 +303,16 @@ the expiration date, and `hasRole` checks if the role is expired by comparing wi
 which should be enough to be considered "permanent". For this reason, it's RECOMMENDED using `type(uint64).max` when
 calling the `grantRole` function to support use cases that require an assignment never to expire.
 
+### Unique and Non-Unique Roles
+
+The standard supports both unique and non-unique roles. Unique roles can be assigned to only one account at a time,
+while non-unique roles can be granted to multiple accounts simultaneously. The roles extension metadata can be used to
+find out if roles are unique, and if the contract does not implement it, all roles should be considered non-unique.
```

Comment:
I think the most common case would be unique roles, instead of non-unique. Shouldn't unique be the default for a nft that not implemented metadata?